### PR TITLE
fix: display name length

### PR DESCRIPTION
**Pull request by sirtorry**
b/172932277

### CODE REVIEW

File: google/cloud/aiplatform/utils.py [hunk 1]
```diff
@@ -194,7 +194,21 @@ def validate_project(project: str) -> bool:
...
-def validate_region(region: str):
+def validate_display_name(display_name: str):
+    """Verify display name is at most 128 chars and UTF-8 only.
+
+    Args:
+        display_name: display name to verify
+    Raises:
+        ValueError: display name is longer than 128 characters
+    """
+    if display_name is None:
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/utils.py:205]: What would be the scenario when this is `None`? Also, if this is the case then the type hint should be `Optional[str]`.

File: google/cloud/aiplatform/utils.py [hunk 2]
```diff
@@ -194,7 +194,21 @@ def validate_project(project: str) -> bool:
     raise ValueError("Please provide a valid project ID")
 
 
-def validate_region(region: str):
+def validate_display_name(display_name: str):
+    """Verify display name is at most 128 chars and UTF-8 only.
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/utils.py:198]: How does this method validate `UTF-8`? If not, I think it's OK to capture in buganizer and TODO here.

File: tests/unit/aiplatform/test_utils.py [hunk 3]
```diff
@@ -193,3 +193,10 @@ def test_full_resource_name_raises_value_error(
             project=project,
             location=location,
         )
+
+
+def test_validate_display_name_raises_value_error():
```
Comment (sasha-gitg) [inline tests/unit/aiplatform/test_utils.py:198]: Should also test the non failure case.

File: google/cloud/aiplatform/datasets.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 
 from google.cloud.aiplatform import base
 from google.cloud.aiplatform import initializer
-from google.cloud.aiplatform.utils import full_resource_name
+from google.cloud.aiplatform import utils
```
Comment (sasha-gitg) [inline google/cloud/aiplatform/datasets.py:25]: Thanks for changing to a module level import!

**Review by sasha-gitg — APPROVED** [review]
A few simple comments. Otherwise LGTM!

File: google/cloud/aiplatform/utils.py (hunk 1 above)
Comment (sirtorry) [reply google/cloud/aiplatform/utils.py]: there are some methods that have optional display names. i've moved the None logic into those methods.

**Pull request merged**